new-lamassu-admin/src/pages/Blacklist/Blacklist.styles.js

```diff
@@ -0,0 +1,42 @@
+export default {
+  grid: {
+    flex: 1,
+    height: '100%'
+  },
+  content: {
+    display: 'flex',
+    flexDirection: 'column',
+    flex: 1,
+    marginLeft: 48
+  },
+  bottomRight: {
+    marginLeft: 'auto',
+    marginTop: 'auto',
+    marginBottom: '24px'
+  },
+  modalTitle: {
+    lineHeight: '120%',
+    color: '#1b2559',
+    fontSize: 14,
+    fontFamily: 'Mont',
+    fontWeight: 900
+  },
+  subtitle: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    flexDirection: 'row'
+  },
+  white: {
+    color: 'white'
```

Comment:
even though it shouldn't be a big deal, we're exporting white as a named variable as well. The reasoning for naming everything instead of using a const is that if we decide to slightly change our base white color the change will be applied to all places if we're using the variable name.